<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CargoManagementWindow</class>
 <widget class="QMainWindow" name="CargoManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>351</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>正在销售</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="salesTableWidget"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="addAllButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>一键添加</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="putButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上架</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="returnpushButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回主界面</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="deleteButton">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>下架</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>371</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>采购清单</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="purchaseTableWidget"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="updatepushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
